pantheon-bundle/src/main/java/com/redhat/pantheon/servlet/module/PublishDraftVersion.java

```diff
@@ -58,6 +63,14 @@ public void run(SlingHttpServletRequest request, PostResponse response, SlingPos
             Locale locale = getLocale(request);
             Module module = getModule(request);
             ModuleLocale moduleLocale = module.getModuleLocale(locale);
+
+
+            //FIXME - this is a hack that needs to be removed when we have the attribute placeholder logic implemented
+            Optional<ModuleVersion> versionToRelease = getModule(request).getReleasedVersion(getLocale(request));
```

Comment:
seems like we already have fetched some of those values:
```suggestion
            Optional<ModuleVersion> versionToRelease = module.getReleasedVersion(locale);
```